doc.go

```diff
@@ -76,4 +76,11 @@
 // supports Prometheus's text and protocol buffer exposition formats. Examples
 // of both push and pull integration are included in the documentation for the
 // root struct's Push and ServeHTTP methods.
+//
+// See Also
+//
+// If you're unfamiliar with Tally and Prometheus, you may want to consult
+// their documentation:
+//   https://godoc.org/github.com/uber-go/tally
+//   https://godoc.org/github.com/prometheus/client_golang/prometheus
```

Comment:
Rather than indenting this, you can actually just list one link per line. godoc renders that as you would expect.

```
// See Also
//
// If you're [..]
//
// https://godoc.org/github.com/uber-go/tally
// https://godoc.org/github.com/...
```

Reply:
Huh. That doesn't work in my local `godoc` invocation :/

Reply:
Weird. Don't worry about it if it doesn't work. :man_shrugging: